reactor-netty-http/src/test/java/reactor/netty/http/client/HttpClientWithTomcatTest.java

```diff
@@ -82,9 +79,9 @@ public void nettyNetChannelAcceptsNettyChannelHandlers() throws Exception {
 		                        .doOnSuccess(v -> latch.countDown())
 		                        .block(Duration.ofSeconds(30));
 
-		assertTrue("Latch didn't time out", latch.await(15, TimeUnit.SECONDS));
-		assertNotNull(response);
-		assertTrue(response.contains("q=test%20d%20dq"));
+		assertThat(latch.await(15, TimeUnit.SECONDS)).as("Latch didn't time out").isTrue();
+		assertThat(response).isNotNull();
+		assertThat(response.contains("q=test%20d%20dq")).isTrue();
```

Comment:
matter of preference but I like to chain assertions when possible:

```suggestion
		assertThat(response)
		    .isNotNull()
		    .contains("q=test%20d%20dq");
```